analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectFunctionalTest.java

```diff
@@ -322,12 +322,12 @@ public void verifyAlignmentReportJson() throws IOException, URISyntaxException {
                 "------------------- project org.acme.gradle:root (path: :)%n" +
                         "\tProject version : 1.0.1 --> 1.0.1.redhat-00002%n" +
                         "%n" +
-                        "\tDependencies : com.yammer.metrics:metrics-core:2.2.0 --> com.yammer.metrics:metrics-core:2.2.0-redhat-00001%n"
-                        +
                         "\tDependencies : org.hibernate:hibernate-core:5.3.7.Final --> org.hibernate:hibernate-core:5.3.7.Final-redhat-00001%n"
                         +
                         "\tDependencies : io.undertow:undertow-core:2.0.15.Final --> io.undertow:undertow-core:2.0.15.Final-redhat-00001%n"
                         +
+                        "\tDependencies : com.yammer.metrics:metrics-core:2.2.0 --> com.yammer.metrics:metrics-core:2.2.0-redhat-00001%n"
+                        +
```

Comment:
Is this change due to it being sorted now?

Reply:
Yes, the order is not sorted, but _maintained_ from the original order.

Here, original usually means as returned from a Gradle API. We'd then lose this order by putting the items into a `HashSet`.

I didn't try to change everywhere, only cases where we printed or checked the order, so that the order should be predictable now.

Reply:
Ok sure.

Reply:
I know  we are currently using a `TreeMap` to sort something alphabetically. I think it's the project names.

If you'd rather sort all output alphabetically, I think I'd still rather store internally in Gradle's order (which I think is in encountered order, like a `List` would do), then use `sorted()` when we go to print it. Although, I am not proposing to change it in this PR.